src/mlpack/methods/ann/layer/elu.hpp

```diff
@@ -62,13 +136,25 @@ class ELU
 {
  public:
   /**
-   * Create the ELU object using the specified parameters. The non zero
+   * Create the ELU object.
+   *
+   * NOTE: Use this constructor for SELU activation function.
```

Comment:
See my comment above. Also, should we add a alias at the end called SELU something like : ``using SELU = ELU<arma::mat, arma::mat>;``.